DataFormats/METReco/interface/BeamHaloSummary.h

```diff
@@ -70,6 +73,12 @@ namespace reco {
     std::vector<int>& GetGlobaliPhiSuspects() { return GlobaliPhiSuspects ;}
     const std::vector<int>& GetGlobaliPhiSuspects() const { return GlobaliPhiSuspects ;}
 
+    std::vector<std::vector<std::pair<char, CaloTowerDetId> > >& GetProblematicStrips() { return ProblematicStrips ;}
```

Comment:
Instead of using a plain ```vector<vector``` a solution optimal for I/O is to linearize the 2D vector and save 1D vector + a vector of indices to point to start/end ranges of the data format.
I'm checking how large these things are on some data to see if this optimization is so significant.